ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -980,6 +980,43 @@ def test_tensorflow_gather_nd(
     )
 
 
+# tile
+@handle_frontend_test(
+    fn_tree="tensorflow.tile",
+    dtypes_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric", full="True")),
```

Comment:
`full` is a bool argument so you should set it equal to `True`, not `"True"`. However in this case that's the default value so you wouldn't need to set it at all.